packages/data/src/options/action-types.js

```diff
@@ -0,0 +1,9 @@
+const TYPES = {
+	RECIEVE_OPTIONS: 'RECIEVE_OPTIONS',
```

Comment:
```suggestion
	RECEIVE_OPTIONS: 'RECEIVE_OPTIONS',
```

Reply:
Fixed in https://github.com/woocommerce/woocommerce-admin/pull/4144/commits/d7061c170d97d703750f0cf53d7cacebc92b9dea